src/webrtc.ts

```diff
@@ -508,37 +513,67 @@ export class WebrtcExtension extends EventEmitter {
     this.logger.info('sent jingle retract', { sessionId, conversationId: session.propose.conversationId });
   }
 
-  async refreshIceServers (): Promise<any[]> {
+  async refreshIceServers (): Promise<ExtendedRTCIceServer[]> {
```

Comment:
I can't add a comment in the place I want, but do you think we should log an explicit error in `on.('connected'...)` if the `refreshIceServers()` fails?